gradle.properties

```diff
@@ -9,7 +9,7 @@
 #
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx8192m
+org.gradle.jvmargs=-Xmx6144m -XX:MaxPermSize=512m -XX:MaxMetaspaceSize=512m
```

Comment:
I wanted to give it as much as possible (and originally set the 8 GB for only myself), but I reduced it because GitHub runners only have 7 GB of RAM. But you have a point, we shouldn't expect that all contributors have so much RAM - but I'd advocate for 4 GB instead.

Reply:
Oh, and the metaspacesize is from the official Gradle Docs, so I added it - I'd suggest adding that to the SDK as well.

Reply:
The SDK uses hundreds of Kotlin data classes and compiles fine with 2gb, I don't think 4gb is necessary. We can always increase is later if desired though